webpack.config.js

```diff
@@ -1,19 +1,28 @@
-const path = require('path')
+const path = require("path");
```

Comment:
unintentional?

Reply:
`prettier` decided this.  Maybe I should exclude this file from Prettier?

Reply:
I'll revert for now... no need for extra churn.